File: output/golang/models/struct/DeviceInformation.go
package main

type DeviceInformation struct {
	PlatformType *string `json:"platform_type" bson:"platform_type"`
	OsVersion *string `json:"os_version" bson:"os_version"`
	OsName *string `json:"os_name" bson:"os_name"`
	InfoMap map[string]string `json:"info_map" bson:"info_map"`
}

File: output/kotlin/models/serializable_data_class/DeviceInformation.kt
package data_class

@Serializable
data class DeviceInformation(
    @SerialName("platform_type") val platformType: String,
    @SerialName("os_version") val osVersion: String?,
    @SerialName("os_name") val osName: String,
    @SerialName("info_map") val infoMap: Map<String, String>,
)